compose.py

```diff
@@ -92,6 +92,10 @@ def start_dockerfile(ddir, images, argmap, dockerfile_name, backends):
 def add_requested_backends(ddir, dockerfile_name, backends):
     df = "# Copying over backends \n"
     for backend in backends:
+        if backend == 'openvino':
+            import build
+            ver = next(iter(build.TRITON_VERSION_MAP.values()))
```

Comment:
For my understanding.. why is this better than `ver = build.TRITON_VERSION_MAP.values()[4][0]`? Does `ver` need to be an iterator?

Reply:
`dict.values()` in python 3.x returns `dict_values` object which is not subscriptable. You can read more about dict-views from [here](https://docs.python.org/3.7/library/stdtypes.html?highlight=dict%20values#dict-views).